### DISCUSSION TITLE
Project 5 - Question

### DISCUSSION THREAD

**Discussion by rachael-clark** [IBAHCM/RPiR#479] [Archive] [answered]
Hi @richardreeve and @soniamitchell 

I was wondering if it would be possible to get an explanation for the task for project 5 again? I am getting myself a little confused about what's being asked of me and hoping that maybe hearing it in a different way might clear that up? - Specifically regarding what it is exactly the function is expected to do - I believe I understand what I am do to with the results in terms of a comparison once I've understood the subsampling. 

Thank you!

**Comment by richardreeve** [comment, marked as answer]
@rachael-clark - I'll try! `sample_by_individuals()` should keep all of the species in the vector of counts (and unlike `sample_by_species()`, etc, there are no subcommunities because this function works on vectors like `tree.pop`). Although the total number of individuals in `tree.pop` is 222,715, if you sample it by individuals you should just drop some/loads of those individuals, retaining just the number specified. Say I ask you to sample just *one* individual, then you'll end up with zeros for all but one of the species, and one of that species (most likely one of the most common species). Conversely if I try to sample 222,715 individuals, then I should get back roughly (or ideally exactly) what we put in. Does that make sense / help?

**Reply by rachael-clark** [reply]
@richardreeve - Okay yes that helps - thank you! So should the length of my subsampled vector always be 299 (assuming I am using the tree.pop), however the sum of the vector should equal the sample size I have asked for? I think this might be where I am tripping myself up as I am not entirely what the output of the function should look like so not sure if its right or wrong?

**Reply by richardreeve** [reply]
Yes and yes.

**Reply by rachael-clark** [reply]
Lovely stuff! Thank you for clarifying that for me
